README.md

```diff
@@ -1,2 +1,66 @@
-# sports-app-react-fe
-This is an intern practice project.
+# Sports App
+
+This is an intern practice project. The goal is to create a basic React + Typescript app for the auth flow.
+
+### Install Dependencies:
+
+```bash
+npm install
+```
+
+### Build the Project
+
+```bash
+npm run build
+```
+
+### Run the Project
+
+```bash
+npm start
+```
+
+### Running the App with Docker
+
+To run the app using Docker, follow these steps:
+
+### 1. Build the Docker Image:
+   In the root of the project, use the following command to build the Docker image:
+
+   ```bash
+   docker-compose build
+   ```
+
+### 2. Run the Docker Container:
+   After the image is built, you can start the container using the command:
+
+   ```bash
+   docker-compose up
+   ```
+
+### 3. Access the App:
+   Open your browser and navigate to http://localhost:3000 to see the application running.
+
+### 4. Stop the Docker Container:
+   When you want to stop the app, use the following command:
+
+   ```bash
+   docker compose down
+   ```
```

Comment:
You still have the code from the Docker changes here.

Reply:
You shouldn't have any changes in this file on this branch, unless it's formatting changes.